src/test/java/it/gov/pagopa/pu/debtpositions/mapper/InstallmentMapperTest.java

```diff
@@ -67,6 +67,23 @@ void givenMapToDtoThenOk(){
     checkNotNullFields(result, "debtor");
```

Comment:
can you check if the syncStatus inner fields are not null (checkNotNullFields on the syncStatus object)?